modules/turnos/institucion.routes.ts

```diff
@@ -0,0 +1,27 @@
+import { institucion, IInstitucion } from './institucion.schema';
+import { MongoQuery, ResourceBase } from '@andes/core';
+
+class InstitucionResource extends ResourceBase<IInstitucion>  {
+    Model = institucion;
+    resourceName = 'institucion';
+    keyId = '_id';
+    searchFileds = {
+        _id: MongoQuery.partialString,
+        nombre: MongoQuery.partialString,
+        detalle: MongoQuery.partialString,
+        tipo: MongoQuery.partialString,
+        customFieldContacto: {
```

Comment:
Aca se te paso el copy/paste. El nombre del campo debería ser contacto. Idem en dirección.
```suggestion
        contacto: {
```

Reply:
Mala mia. Corregido.

Reply:
corregido